WeaponSwingTimer_Hunter.lua

```diff
@@ -187,33 +186,41 @@ end
 
 addon_data.hunter.UpdateRangeCastSpeedModifier = function()
     local speed = 1.0
-    for i=1, 40 do
-        name, _ = UnitAura("player", i)
-        if name == "Quick Shots" then
+    local buffs = {3045, 26635, 6150, 28866, 12889} -- aura ids for each haste effect
+	for i=1, 40 do
+        local _, _, _, _, _, _, _, _, _, buffId = UnitBuff("player",i)
```

Comment:
UnitAura seems better here, and use space not tabs.

Reply:
I'm not the author of the changes, I only added WatchYourSixx's changes since he made his own repo instead of forking - this is only to make adding it to the main addon easier for comparison. I will try to respond as best I can though. BuffIDs are correct for classic. Not all IDs are the same for classic and retail, since some are removed in retail. I suspect IDs were used instead of name to avoid localization issues. UnitAura does seem better here TBH, due to CoT.